source/lib.ts

```diff
@@ -136,12 +171,13 @@ const parseOptions = (passedOptions: Partial<Options>): Configuration => {
 			response.statusCode < 400,
 		skip: (_request: Request, _response: Response): boolean => false,
 		keyGenerator(request: Request, _response: Response): string {
-			if (!request.ip) {
-				console.error(
```

Comment:
Maybe we should stick with `console.error` instead of `console.warn` for consistency's sake. 

I originally chose `warn` to make it distinct from the `Throw` option before we got rid of that, but I had also forgotten that we had this check that did a `console.error`.

Reply:
Okay, will change to `console.error`